src/registrar/templates/emails/status_change_rejected.txt

```diff
@@ -8,9 +8,65 @@ REQUEST RECEIVED ON: {{ application.submission_date|date }}
 STATUS: Rejected
 
 ----------------------------------------------------------------
+{% if application.rejection_reason != 'other' %}
```

Comment:
A little confused about why we want rejection_reason == 'other' to hide this text...but I am sure it was a requirement somewhere, so I will just note my confusion and let you carry on.
NO CHANGE NEEDED